### DISCUSSION TITLE
Inspect external network requests?

### DISCUSSION THREAD

**Discussion by timothymalcham** [remix-run/remix#3298] [Proposals]
In my remix app, I need to use an external REST API. I call the endpoints using fetch in my actions/loaders. When I look in the dev tools to inspect network requests, these calls to the endpoints don't show up (obviously, since they happen on the server). 

Would it be possible to have some sort of mechanism to easily inspect network requests that happen server-side? I'm not much of a node expert or developer, so it's entirely likely there's some middleware available for this, but I'm also unsure where that would go since much of the server is abstracted away with remix (a good thing!). 

It would be nice to see things like, how long did it take for this external endpoint to return a response, etc.

**Comment by sergiodxa** [comment]
You can setup Node.js built-in inspector to see requests, the setup should be similar to any other Node.js app.
